components/Blog/styling/Blog.styles.ts

```diff
@@ -0,0 +1,61 @@
+import styled from 'styled-components';
+
+import {
+  LargeParagraph,
+  H2,
+  H3,
+  DefaultParagraph,
+} from 'components/atoms/typography.styles';
+
+export const TopDescription = styled(LargeParagraph)`
+  padding-bottom: 50px !important;
+  font-style: italic;
+
+  @media screen and (min-width: 768px) {
+    font-size: 36px;
+    line-height: 120%;
+    font-weight: 400;
+    padding-bottom: 55px !important;
+  }
+`;
+
+export const MainTitle = styled(H2)`
+  padding-bottom: 20px !important;
+
+  @media screen and (min-width: 768px) {
+    font-size: 48px;
+    line-height: 57.6px;
```

Comment:
Do these line heights come from figma ? There are % a x.y px values here, doesn't feel consistent.

Reply:
Fair enough. Actually it is not hard to see that it is the 120% of the font size.